packages/elements/src/react/common/form/index.tsx

```diff
@@ -306,10 +332,19 @@ const useInput = ({
     };
   }
 
+  if (isSupported) {
+    props = {
+      autoComplete: 'webauthn',
+    };
+  }
+
   // Filter out invalid props that should not be passed through
   // @ts-expect-error - Doesn't know about type narrowing by type here
   const { validatePassword: _1, ...rest } = passthroughProps;
 
+  // @ts-expect-error - Depending on type the props can be different
+  delete rest['passkeyAutofill'];
```

Comment:
Do we have a better way for sanitize props ?

Reply:
Directly above it, you could move that there